build/lib/src/resource/resource.dart

```diff
@@ -23,10 +24,16 @@ class Resource<T> {
   /// disposed.
   final DisposeInstance<T> _userDispose;
 
+  /// Optional method which is called before the process is going to exit. This
```

Comment:
[nit] blank line following first sentence. Similar in other doc comments.

Reply:
done